README.md

```diff
@@ -27,70 +27,65 @@ In the [`bkc`](https://github.com/intel/ccc-linux-guest-hardening/tree/master/bk
+  and VM-based snapshot fuzzing. This setup does not run inside a VM or container!
 
-The userspace installation and fuzzing workflow has been tested for recent
-Ubuntu (>=20.04) and Debian (>=bullseye). It only requires Python3:
+- **Recent Debian/Ubuntu:** The userspace installation and fuzzing workflow has
+  been tested for recent Ubuntu (>=20.04) and Debian (>=bullseye).
 
-- `python3`
-- `python3-venv`
-
-~~~
-sudo apt-get install python3 python3-venv
-~~~
+- **Know your Kernel:** Working knowledge of Linux console, kernel build and boot,
+  and an idea of the kernel version and feature you want to test.
 
 ## Installation
 
-Clone this repo to a new directory, e.g. `~/tdx`:
-
-```shell
-git clone https://github.com/intel/ccc-linux-guest-hardening ~/tdx
-cd ~/tdx
-```
-
-We use Ansible playbooks to support local or remote installation.
-
-### Local
-
-- installation directory: `<repo_root>/`
+### The installation and the fuzzing runtime requires Python3 and the virtual environment package:
 
-Run the deployment with:
 ~~~
-make deploy
+sudo apt-get install python3 python3-venv
 ~~~
 
-You will be prompted for your root password.
-If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
+### Clone this repo to a new top-level workspace and install using `make deploy`:
 
-### Remote
+  ```bash
+  git clone https://github.com/intel/ccc-linux-guest-hardening ~/cocofuzz
+  cd ~/cocofuzz
+  make deploy
+  ```
 
-- installation directory: `$HOME/ccc`
+**Note:** The installation uses [Ansible](https://docs.ansible.com/ansible/latest/).
+The main system modification is to install a patched host kernel (`.deb` package)
+and fixing the `grub` config to make it boot. Ansible will also add the current
+user to group `kvm` and pull in a few build dependencies and tools via `apt`. 
+The rest of the stack consists of userspace tools and scripts which are only 
+available in a local Python virtual environment.
 
-You will have to update the `deploy/inventory` file to describe your nodes, according to [Ansible's inventory guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
-Make sure to **remove** the first line:
+### If not yet done, reboot to launch the kAFL/SDV emulation kernel:
 
-~~~
-localhost ansible_connection=local
-~~~
+```bash
+uname -a
+# Linux tdx-fuzz0 5.6.0-rc1-tdfl+ #15 SMP Wed May 25 02:23:44 CEST 2022 x86_64 x86_64 x86_64 GNU/Linux
+```
 
-And run the deployment:
+```bash
+dmesg|grep KVM-PT
+# [KVM-PT] Info:  CPU is supported!
+# [KVM-PT] Info:  LVT PMI handler registrated!
+```
 
-~~~
-make deploy
-~~~
 
-Note: if your nodes require a proxy setup, update the `group_vars/all.yml`.
 
 ## Activate the environment
 
 When the installation is complete, you will find several tools and scripts
 inside the installation directory of the target system.
 
+The environment defines various default paths used by multiple layers of
```

Comment:
The reference to `env.sh` has disappeared and this paragraph does not make sense. There was a comment like "You can activate like so: make env", followed by the longer explanation of the env setup.

Reply:
Added the [code block](https://github.com/tz0/ccc-linux-guest-hardening/blob/62e1c0b39085c9cb1904c0293cc2a3d20ebcdef4/README.md?plain=1#L81) of verifying tools, including `env.sh`.  

Swapped the paragraph with the below one. Please point out the original comment if there is any explanation still missing.